packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -359,29 +386,6 @@ export default function ViewList< Item >( props: ViewListProps< Item > ) {
 		[ baseId, getItemId ]
 	);
 
-	const store = useCompositeStore( {
-		defaultActiveId: getItemDomId( selectedItem ),
-	} ) as CompositeStore; // TODO, remove once composite APIs are public
-
-	// Manage focused item, when the active one is removed from the list.
-	const isActiveIdInList = useStoreState(
-		store,
-		( state: { items: any[]; activeId: any } ) =>
-			state.items.some(
-				( item: { id: any } ) => item.id === state.activeId
-			)
-	);
-	useEffect( () => {
-		if ( ! isActiveIdInList ) {
-			// Prefer going down, except if there is no item below (last item), then go up (last item in list).
-			if ( store.down() ) {
-				store.move( store.down() );
-			} else if ( store.up() ) {
-				store.move( store.up() );
-			}
-		}
-	}, [ isActiveIdInList ] );
```

Comment:
This logic was moved to a new `ViewListWrapper` component, that can access the `store` via `Composite.Context`